helm/templates/service/deployment.yaml

```diff
@@ -76,6 +76,8 @@ spec:
               value: {{ .Values.env }}
             - name: SBOMER_ROUTE_HOST
               value: {{ .Values.service.route.host }}
+            - name: SBOMER_SERVICE_URL
+              value: "https://{{ .Values.service.route.host }}"
```

Comment:
Since we have it in the config map, then why not something like this?

```suggestion
              valueFrom:
                configMapKeyRef:
                  name: {{ .Release.Name }}-config
                  key: SBOMER_SERVICE_URL
```

Reply:
ah! I forgot to remove it .... not needed in the service deployment I believe

Reply:
But I guess it should be exposed in Tekton Tasks.